package.json

```diff
@@ -22,6 +23,7 @@
   },
   "homepage": "https://github.com/jesseskinner/svelte-autosize#readme",
   "dependencies": {
-    "autosize": "*"
+    "autosize": "*",
+    "svelte": "^4.0.0"
```

Comment:
Shouldn't `svelte` be a `peerDependency` instead? 👀

Reply:
Yes I think you're right.. I'll make that change. Thanks!
Note that I already changed it to be >3.0.0 so that it's compatible with versions 3, 4 and 5.